WcaOnRails/app/models/user_group.rb

```diff
@@ -40,8 +48,25 @@ def roles
     end
   end
 
+  # TODO: Once the roles migration is done, add a validation to make sure there is only one lead_user per group.
+  def lead_user
```

Comment:
I feel like I saw this in another PR before...

Reply:
Oh is it? I don't think I've added it anywhere before? 🤔 Maybe you saw it in same PR in earlier review?

Reply:
Oh, it is really possible that I lost track of the different PRs while reviewing your work all at once :sweat_smile: